src/helpers/dom/element.js

```diff
@@ -1017,3 +1017,16 @@ export function isInput(element) {
 export function isOutsideInput(element) {
   return isInput(element) && element.className.indexOf('handsontableInput') == -1 && element.className.indexOf('copyPaste') == -1;
 }
+
+/**
+ * Check if the given DOM element to focus.
+ *
+ * @param {HTMLElement} element - DOM element
+ */
+export function checkToFocus(element) {
```

Comment:
That helper name is misleading. It does not only check if an element can be focused it `.select()` an element as well. Maybe the name `selectElementIfAllowed` will be more appropriate?

And don't forget to add tests for this helper :)